tests/nnstreamer_flatbuf/runTest.sh

```diff
@@ -32,6 +32,24 @@ convertBMP2PNG
 
 PATH_TO_PLUGIN="../../build"
 
+if [[ -d $PATH_TO_PLUGIN ]]; then
+    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer/extra"
```

Comment:
plz check the path for flatbuf, flexbuf and protobuf. 
This path (build/ext/nnstreamer/extra) are available when grpc is supported.